WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -1486,10 +1489,31 @@ def getSiteMask( self, siteState = 'Active' ):
     return S_OK( siteList )
 
 #############################################################################
-  def getSiteMaskStatus( self ):
+  def getSiteMaskStatus( self, sites = None ):
     """ Get the currently site mask status
     """
-    cmd = "SELECT Site,Status FROM SiteMask"
+    if isinstance(sites, list):
+
+      cmd = "SELECT Site, Status FROM SiteMask WHERE"
```

Comment:
```SELECT Site, Status From SiteMask WHERE site='site1' OR site='site2'```
is equivalent to 
```SELECT Site, Status From SiteMask WHERE site in ('site1','site2')```
And also works with
```SELECT Site, Status From SiteMask WHERE site in ('site1')```
This instead of the current implementation one could do:
```
if instance( sites, basestring ):
  sites  = [sites]
if instance( sites, list ):
  sitesString = ",".join( "'%s'" % site for site in sites) ## note the single quotes
  cmd = "SELECT Site, Status FROM SiteMask WHERE site in (%s)" % sitesString
...
```
It is just simpler than the concatenation of OR statements